container/kvm/kvm.go

```diff
@@ -184,7 +184,9 @@ func (manager *containerManager) CreateContainer(
 	logger.Tracef("create the container, constraints: %v", cons)
 	if err := kvmContainer.Start(startParams); err != nil {
 		err = errors.Annotate(err, "kvm container creation failed")
-		logger.Infof(err.Error())
+		// Logged as debug here because it is logged as an error in
+		// worker/provisioner.
+		logger.Debugf(err.Error())
```

Comment:
I agree. This was existing code. I just wanted to turn down the amount of logging of the same error.

Removed.